.github/workflows/_reusable_run_tests.yml

```diff
@@ -191,7 +191,7 @@ jobs:
           echo "Building WireFoundation..."
           xcodebuild build-for-testing -workspace wire-ios-mono.xcworkspace -scheme WireFoundation-Package -resultBundlePath xcodebuild-wire-foundation.xcresult -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee xcodebuild-wire-foundation.log | bundle exec xcpretty
 
-      - uses: kronenthaler/analyze-xcoderesults-action@0.1.9
+      - uses: wireapp/analyze-xcoderesults-action@v1
```

Comment:
`kronenthaler/analyze-xcoderesults-action@0.1.9` is broken in Xcode 16 so fixed this with a fork and migrated to that fork.